src/AppBundle/Membership/AdherentChangePasswordHandler.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace AppBundle\Membership;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Security\Core\Encoder\EncoderFactoryInterface;
+
+class AdherentChangePasswordHandler
+{
+    private $manager;
+    private $encoderFactory;
+
+    public function __construct(ObjectManager $manager, EncoderFactoryInterface $encoderFactory)
```

Comment:
Ok, its up to you. Personally I prefer to use the `PasswordEncoder` (available at `security.password_encoder`) because it's just do what you do (so more convenient) :)